server/models/annotation.js

```diff
@@ -15,6 +15,7 @@ const annotationSchema = new Schema({
 
   text: { type: String, trim: true },
   articleText: String,
+  ranges: { type: Array, default: [] },
```

Comment:
We should be able to specify this part of the schema more specifically (assuming each member of the array should have the same format). I can change this and make a commit.